Cows-HacknSlash/Assets/Scripts/NPC/RatBehaviour.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class RatBehaviour : MonoBehaviour {
+
+   
+    [SerializeField]
+    private Transform Eyes;             // Spherecast will release ray from this transfrom.
```

Comment:
Code + comments here on the same line again (don't worry about it, just notifying you)